lib/benchee/formatters/unit/count.ex

```diff
@@ -0,0 +1,50 @@
+defmodule Benchee.Unit.Count do
+  alias Benchee.Unit.Common
+
+  @behaviour Benchee.Unit
+
+  @one_billion 1_000_000_000
+  @one_million 1_000_000
+  @one_thousand 1_000
+
+  @units %{
+    billion:  %{ magnitude: @one_billion, short: "B", long: "Billion"},
+    million:  %{ magnitude: @one_million, short: "M", long: "Million"},
+    thousand: %{ magnitude: @one_thousand, short: "K", long: "Thousand"},
+    one:      %{ magnitude: 1, short: "", long: ""},
+  }
+
+  @doc """
+  Scales a value representing a count in ones into a larger unit if appropriate
+
+  ## Examples
+
+      iex> Benchee.Unit.Count.scale(4_321.09)
+      {4.32109, :thousand}
+
+      iex> Benchee.Unit.Count.scale(0.0045)
+      {0.0045, :one}
```

Comment:
Me too! I was thinking it might make sense to move some other test cases to doctests at some point. What do you think?